packages/venia-concept/_buildpack/create.js

```diff
@@ -34,6 +34,11 @@ function createProjectFromVenia({ fs, tasks, options }) {
         'storybook:build': 'build-storybook -c src/.storybook -o storybook-dist'
     };
 
+    // Dependencies we want to ensure are added to the scaffolded app.
+    const devDependenciesToInsert = {
+        '@magento/venia-sample-backends': '~0.0.1'
```

Comment:
Does this seem OK to folks? I believe the extension was meant to be used for scaffolded apps anyways.

Reply:
Yeah the extension is for new apps created using PWA Studio. This is ok with me.

Reply:
I'm going to say no, unless there is strong evidence that there is a large portion of scaffolded apps using our sample backend. This just seems like unnecessary spam, and we shouldn't annoy developers with a necessary step they have to take to go live that they didn't opt-in to.

Reply:
Compromise: 

Only attach dependency if user has selected one of the sample backends.